Tasks/VsTestPlatformToolInstaller/cieventlogger.ts

```diff
@@ -16,9 +16,12 @@ export function publishEvent(subFeature: string, properties: { [key: string]: an
     try {
         tl.assertAgent('2.125.0');
         properties['subFeature'] = subFeature;
-        tl.publishTelemetry(area, feature, Object.assign(getDefaultProps(), properties));
-
+        try {
+            tl.publishTelemetry(area, feature, Object.assign(getDefaultProps(), properties));
+        } catch (err) {
```

Comment:
this is not required as it's already handled in PublishTelemetry call

Reply:
and if it is not handled this needs to go into that code..

---
In reply to: [180741561](https://github.com/Microsoft/vsts-tasks/pull/6943#discussion_r180741561) [](ancestors = 180741561)

Reply:
Just wanted to be double sure on our end and not rely on the try catch there in case someone changed it without our notice.

Can remove it if you guys feel its not required. I personally thought its safer to leave it around just for the simple fact that the telemetry implementation may change.

Reply:
removed this from here